vere/main.c

```diff
@@ -250,12 +258,47 @@ _main_getopt(c3_i argc, c3_c** argv)
   if ( u3_Host.ops_u.nuu != c3y && u3_Host.ops_u.url_c != 0 ) {
     fprintf(stderr, "-u only makes sense when bootstrapping a new instance\n");
     return c3n;
+  }
+
+  if ( u3_Host.ops_u.nuu != c3y && u3_Host.ops_u.sap_c != 0 ) {
+    fprintf(stderr, "-m only makes sense when bootstrapping a new instance\n");
+    return c3n;
+  }
+
+  if ( u3_Host.ops_u.fak_c != 0 && u3_Host.ops_u.sap_c != 0 ) {
+    fprintf(stderr, "-m and -F cannot be used together\n");
+    return c3n;
+  }
+
+  if ( u3_Host.ops_u.ets_c != 0 && u3_Host.ops_u.sap_c != 0 ) {
+    fprintf(stderr, "-m and -e cannot be used together\n");
```

Comment:
Is this necessary? It seems plausible that someone could start from an explicit snapshot, and then continue from the specified ethereum gateway.

Reply:
Fixed